core/ipc/sink/aws-sqs-impl/src/main/java/org/opennms/core/ipc/sink/aws/sqs/AwsConditionalSinkContext.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2017 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2017 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.core.ipc.sink.aws.sqs;
+
+import static org.opennms.core.ipc.sink.common.SinkStrategy.Strategy.AWS;
+
+import org.opennms.core.ipc.sink.api.MessageConsumerManager;
+import org.opennms.core.ipc.sink.common.SinkStrategy;
+import org.opennms.core.logging.Logging;
+import org.opennms.core.logging.Logging.MDCCloseable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ConfigurationCondition;
+import org.springframework.context.annotation.ImportResource;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+/**
+ * The Class AwsConditionalSinkContext.
+ * 
+ * @author <a href="mailto:agalue@opennms.org">Alejandro Galue</a>
+ */
+@Configuration
+@Conditional(AwsConditionalSinkContext.Condition.class)
+@ImportResource("/META-INF/opennms/applicationContext-ipc-sink-server-aws.xml")
+public class AwsConditionalSinkContext {
```

Comment:
Can we prefix these classes with "Sqs" instead of "Aws"?

Reply:
Just "Sqs", or maybe "AwsSqs" ? I mean, if just "Aws" doesn't look right, just "Sqs" is meaningless as well.

Reply:
"AwsSqs" is pretty unreadable to most people, I'd prefer "AmazonSQS" for clarity.